src/connector/src/parser/upsert_parser.rs

```diff
@@ -50,21 +51,37 @@ async fn build_accessor_builder(
     }
 }
 
+fn check_rw_kafka_key(columns: &Vec<SourceColumnDesc>) -> bool {
+    for col in columns {
+        if col.name.starts_with("_rw_kafka_key") {
```

Comment:
down vote for hard encoding

Reply:
I think maybe we will have more and more special columns, check them with the name might be a good idea.